### ISSUE TITLE
ClientStream - failed to reconnect

### ISSUE THREAD

**Issue by deankwok** [oceanbase/oblogclient#1]
Any possible cause? Thanks in advance. :)  The following code: 

BTW, whether include the detail document for oblogclient in java

```
16:56:24.038 [Thread-0] WARN com.oceanbase.clogproxy.client.connection.ClientStream - start to reconnect...
16:56:24.112 [Thread-0] DEBUG io.netty.util.internal.logging.InternalLoggerFactory - Using SLF4J as the default logging framework
16:56:24.116 [Thread-0] DEBUG io.netty.util.internal.PlatformDependent0 - -Dio.netty.noUnsafe: false
16:56:24.116 [Thread-0] DEBUG io.netty.util.internal.PlatformDependent0 - Java version: 8
16:56:24.117 [Thread-0] DEBUG io.netty.util.internal.PlatformDependent0 - sun.misc.Unsafe.theUnsafe: available
16:56:24.118 [Thread-0] DEBUG io.netty.util.internal.PlatformDependent0 - sun.misc.Unsafe.copyMemory: available
16:56:24.118 [Thread-0] DEBUG io.netty.util.internal.PlatformDependent0 - java.nio.Buffer.address: available
16:56:24.119 [Thread-0] DEBUG io.netty.util.internal.PlatformDependent0 - direct buffer constructor: available
16:56:24.120 [Thread-0] DEBUG io.netty.util.internal.PlatformDependent0 - java.nio.Bits.unaligned: available, true
16:56:24.120 [Thread-0] DEBUG io.netty.util.internal.PlatformDependent0 - jdk.internal.misc.Unsafe.allocateUninitializedArray(int): unavailable prior to Java9
16:56:24.120 [Thread-0] DEBUG io.netty.util.internal.PlatformDependent0 - java.nio.DirectByteBuffer.<init>(long, int): available
16:56:24.120 [Thread-0] DEBUG io.netty.util.internal.PlatformDependent - sun.misc.Unsafe: available
16:56:24.120 [Thread-0] DEBUG io.netty.util.internal.PlatformDependent - -Dio.netty.tmpdir: /tmp (java.io.tmpdir)
16:56:24.121 [Thread-0] DEBUG io.netty.util.internal.PlatformDependent - -Dio.netty.bitMode: 64 (sun.arch.data.model)
16:56:24.123 [Thread-0] DEBUG io.netty.util.internal.PlatformDependent - -Dio.netty.maxDirectMemory: 3717201920 bytes
16:56:24.123 [Thread-0] DEBUG io.netty.util.internal.PlatformDependent - -Dio.netty.uninitializedArrayAllocationThreshold: -1
16:56:24.124 [Thread-0] DEBUG io.netty.util.internal.CleanerJava6 - java.nio.ByteBuffer.cleaner(): available
16:56:24.124 [Thread-0] DEBUG io.netty.util.internal.PlatformDependent - -Dio.netty.noPreferDirect: false
16:56:24.138 [Thread-0] DEBUG io.netty.util.internal.NativeLibraryLoader - -Dio.netty.native.workdir: /tmp (io.netty.tmpdir)
16:56:24.138 [Thread-0] DEBUG io.netty.util.internal.NativeLibraryLoader - -Dio.netty.native.deleteLibAfterLoading: true
16:56:24.139 [Thread-0] DEBUG io.netty.util.internal.NativeLibraryLoader - -Dio.netty.native.tryPatchShadedId: true
16:56:24.151 [Thread-0] DEBUG io.netty.util.internal.NativeLibraryLoader - Successfully loaded the library /tmp/libnetty_transport_native_epoll_x86_641612895115132305606.so
16:56:24.155 [Thread-0] DEBUG io.netty.util.NetUtil - -Djava.net.preferIPv4Stack: false
16:56:24.155 [Thread-0] DEBUG io.netty.util.NetUtil - -Djava.net.preferIPv6Addresses: false
16:56:24.157 [Thread-0] DEBUG io.netty.util.NetUtilInitializations - Loopback interface: lo (lo, 0:0:0:0:0:0:0:1%lo)
16:56:24.158 [Thread-0] DEBUG io.netty.util.NetUtil - /proc/sys/net/core/somaxconn: 4096
16:56:24.163 [Thread-0] DEBUG io.netty.channel.MultithreadEventLoopGroup - -Dio.netty.eventLoopThreads: 32
16:56:24.179 [Thread-0] DEBUG io.netty.util.internal.InternalThreadLocalMap - -Dio.netty.threadLocalMap.stringBuilder.initialSize: 1024
16:56:24.179 [Thread-0] DEBUG io.netty.util.internal.InternalThreadLocalMap - -Dio.netty.threadLocalMap.stringBuilder.maxSize: 4096
16:56:24.191 [Thread-0] DEBUG io.netty.util.internal.PlatformDependent - org.jctools-core.MpscChunkedArrayQueue: available
16:56:24.221 [Thread-0] DEBUG io.netty.channel.DefaultChannelId - -Dio.netty.processId: 953696 (auto-detected)
16:56:24.224 [Thread-0] DEBUG io.netty.channel.DefaultChannelId - -Dio.netty.machineId: 2c:f0:5d:ff:fe:3a:44:eb (auto-detected)
16:56:24.235 [Thread-0] DEBUG io.netty.util.ResourceLeakDetector - -Dio.netty.leakDetection.level: simple
16:56:24.235 [Thread-0] DEBUG io.netty.util.ResourceLeakDetector - -Dio.netty.leakDetection.targetRecords: 4
16:56:24.254 [Thread-0] DEBUG io.netty.buffer.PooledByteBufAllocator - -Dio.netty.allocator.numHeapArenas: 32
16:56:24.255 [Thread-0] DEBUG io.netty.buffer.PooledByteBufAllocator - -Dio.netty.allocator.numDirectArenas: 32
16:56:24.255 [Thread-0] DEBUG io.netty.buffer.PooledByteBufAllocator - -Dio.netty.allocator.pageSize: 8192
16:56:24.255 [Thread-0] DEBUG io.netty.buffer.PooledByteBufAllocator - -Dio.netty.allocator.maxOrder: 11
16:56:24.255 [Thread-0] DEBUG io.netty.buffer.PooledByteBufAllocator - -Dio.netty.allocator.chunkSize: 16777216
16:56:24.255 [Thread-0] DEBUG io.netty.buffer.PooledByteBufAllocator - -Dio.netty.allocator.smallCacheSize: 256
16:56:24.255 [Thread-0] DEBUG io.netty.buffer.PooledByteBufAllocator - -Dio.netty.allocator.normalCacheSize: 64
16:56:24.255 [Thread-0] DEBUG io.netty.buffer.PooledByteBufAllocator - -Dio.netty.allocator.maxCachedBufferCapacity: 32768
16:56:24.255 [Thread-0] DEBUG io.netty.buffer.PooledByteBufAllocator - -Dio.netty.allocator.cacheTrimInterval: 8192
16:56:24.255 [Thread-0] DEBUG io.netty.buffer.PooledByteBufAllocator - -Dio.netty.allocator.cacheTrimIntervalMillis: 0
16:56:24.255 [Thread-0] DEBUG io.netty.buffer.PooledByteBufAllocator - -Dio.netty.allocator.useCacheForAllThreads: true
16:56:24.255 [Thread-0] DEBUG io.netty.buffer.PooledByteBufAllocator - -Dio.netty.allocator.maxCachedByteBuffersPerChunk: 1023
16:56:24.267 [Thread-0] DEBUG io.netty.buffer.ByteBufUtil - -Dio.netty.allocator.type: pooled
16:56:24.267 [Thread-0] DEBUG io.netty.buffer.ByteBufUtil - -Dio.netty.threadLocalDirectBufferSize: 0
16:56:24.267 [Thread-0] DEBUG io.netty.buffer.ByteBufUtil - -Dio.netty.maxThreadLocalCharBufferSize: 16384
16:56:24.337 [Thread-0] ERROR com.oceanbase.clogproxy.client.connection.ClientStream - failed to reconnect, retry count: 1, max: -1
16:56:26.338 [Thread-0] WARN com.oceanbase.clogproxy.client.connection.ClientStream - start to reconnect...
16:56:26.339 [Thread-0] ERROR com.oceanbase.clogproxy.client.connection.ClientStream - failed to reconnect, retry count: 2, max: -1
16:56:28.339 [Thread-0] WARN com.oceanbase.clogproxy.client.connection.ClientStream - start to reconnect...
16:56:28.341 [Thread-0] ERROR com.oceanbase.clogproxy.client.connection.ClientStream - failed to reconnect, retry count: 3, max: -1
16:56:30.341 [Thread-0] WARN com.oceanbase.clogproxy.client.connection.ClientStream - start to reconnect...
16:56:30.342 [Thread-0] ERROR com.oceanbase.clogproxy.client.connection.ClientStream - failed to reconnect, retry count: 4, max: -1
```

**Comment by whhe** [comment]
Looks like the connection is not created successfully. I have added the exception message of `reconnect` method into logger and push the code to snapshot repo with version `1.0.1-SNAPSHOT`. You can use the newest code to find the cause.

**Comment by whhe** [comment]
The patch about logger update has been released in version `1.0.1`.
